argo-cd-apps/base/member/infra-deployments/pipeline-service/pipeline-service.yaml

```diff
@@ -29,18 +29,18 @@ spec:
     spec:
       project: default
       source:
-        path: '{{values.sourceRoot}}/{{values.environment}}/{{values.clusterDir}}'
+        path: "{{values.sourceRoot}}/{{values.environment}}/{{values.clusterDir}}"
```

Comment:
why are we touching this file here?

Reply:
Seem like auto-formatting of my IDE, will revert